tests/test_autodoc.py

```diff
@@ -949,7 +949,7 @@ def call_autodoc(objtype, name):
         '      Update state of cell to *state*.',
         '      ',
     ]
-    if sys.version_info < (3, 5):
+    if sys.version_info < (3, 5, 4):
```

Comment:
I cherry-pick'ed #4556 manually from master branch. It should be also merged to 1.7 branch.